.github/workflows/CI-CD.yml

```diff
@@ -0,0 +1,47 @@
+name: IdlImporter CI/CD
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with:
+        fetch-depth: 0
+
+    - name: Install .NET Core
+      uses: actions/setup-dotnet@77a48bbb85cd3809790223e895ba4b6cbc110fe8
+      with:
+        dotnet-version: 6.0.x
+
+    - name: Restore
+      run: dotnet restore
+
+    - name: Build
+      run: dotnet build
+
+    - name: Pack
+      run: dotnet pack IDLImporter --include-symbols --no-restore --no-build -o .
+
+    - name: Pack
+      run: dotnet pack IdlImpTool --include-symbols --no-restore --no-build -o .
+
+    - name: Publish to Nuget
+      run: dotnet nuget push *.nupkg -s https://api.nuget.org/v3/index.json -k ${{secrets.SILLSDEV_PUBLISH_NUGET_ORG}} --skip-duplicate
```

Comment:
I think we should also push (and publish below) the *.snupkg files

Reply:
As you can see in the log for the liblcm action the symbol packages get pushed as part of this command unless you give a flag to skip them.
https://github.com/sillsdev/liblcm/runs/7617839291?check_suite_focus=true
Are you sure we need them in the artifacts? I can add that if you promise to use them, I know I never would. 😄